src/GitHub/Data/Teams.hs

```diff
@@ -83,8 +83,8 @@ data CreateTeam = CreateTeam
     { createTeamName        :: !(Name Team)
     , createTeamDescription :: !(Maybe Text)
     , createTeamRepoNames   :: !(Vector (Name Repo))
-    -- , createTeamPrivacy    :: Privacy
-    , createTeamPermission  :: Permission
+    , createTeamPrivacy     :: !(Maybe Privacy)
```

Comment:
I think the reasoning in specifying privacy in *create* is that you have to think about it, and explicitly specify it, and not rely on the default on GitHub backend.

The edit `Maybe` says "don't change it", which is different, and arguably good thing. I don't know how `permission` works; the docs are ambiguous. If they can be left out when editing, that would be good improvement too.